compconfig/compconfig/functions.py

```diff
@@ -55,11 +69,15 @@ def run_model(meta_param_dict, adjustment):
     '''
     meta_params = MetaParams()
     meta_params.adjust(meta_param_dict)
-    params = Specifications(year=meta_params.start_year)
+    if meta_params.data_source == "PUF":
+        data = retrieve_puf(AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY)
+    else:
+        data = "cps"
+    params = Specifications(year=meta_params.year, call_tc=True, data=data)
```

Comment:
I think this needs to be converted to the `int` type via [`np.tolist`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.tolist.html):
```suggestion
    params = Specifications(year=meta_params.year.tolist(), call_tc=True, data=data)
```